docs/source/conventions.rst

```diff
@@ -0,0 +1,239 @@
+
+* ``ground-rydberg``: :math:`|r\rangle = (1, 0)^T,~~|g\rangle = (0, 1)^T`
+* ``digital``: :math:`|g\rangle = (1, 0)^T,~~|h\rangle = (0, 1)^T`
+
+On the other hand, the ``XY`` basis uses an independent set of qubit states that are 
+labelled :math:`|0\rangle` and :math:`|1\rangle` and follow the standard convention:
+
+* ``XY``: :math:`|0\rangle = (1, 0)^T,~~|1\rangle = (0, 1)^T`
+
+Multi-partite states
+*************************
+
+The combined quantum state of multiple atoms respects their order in the ``Register``.
+For a register with ordered atoms ``(q0, q1, q2, ..., qn)``, the full quantum state will be
+
+.. math:: |q_0, q_1, q_2, ...\rangle = |q_0\rangle \otimes |q_1\rangle \otimes |q_2\rangle \otimes ... \otimes |q_n\rangle
+
+.. note::
+  The atoms may be labelled arbitrarily without any inherent order, it's only the
+  order with which they are stored in the ``Register`` (as returned by 
+  ``Register.qubit_ids``) that matters .
+
+.. _SPAM:
+
+State Preparation and Measurement
+####################################
+
+.. list-table:: Initial State and Measurement Conventions
+   :align: center
+   :widths: 60 40 75
+   :header-rows: 1
+
+   * - Basis
+     - Initial state
+     - Measurement
+   * - ``ground-rydberg``
+     - :math:`|g\rangle`
+     - |
+       | :math:`|r\rangle \rightarrow 1`
+       | :math:`|g\rangle,|h\rangle \rightarrow 0`
+   * - ``digital``
+     - :math:`|g\rangle`
+     - |
+       | :math:`|h\rangle \rightarrow 1`
+       | :math:`|g\rangle,|r\rangle \rightarrow 0`
+   * - ``XY``
+     - :math:`|0\rangle`
+     - |
+       | :math:`|1\rangle \rightarrow 1`
+       | :math:`|0\rangle \rightarrow 0`
+
+Measurement samples order
+***************************
+
+Measurement samples are returned as a sequence of 0s and 1s, in
+the same order as the atoms in the ``Register`` and in the multi-partite state.
+
+For example, a four-qutrit state :math:`|q_0, q_1, q_2, q_3\rangle` that's
+projected onto :math:`|g, r, h, r\rangle` when measured will record a count to
+sample
+
+* ``0101``, if measured in the ``ground-rydberg`` basis
+* ``0010``, if measured in the ``digital`` basis
+
+Hamiltonians
+####################################
+
+Independently of the mode of operation, the Hamiltonian describing the system
+can be written as
+
+.. math:: H(t) = \sum_i \left (H^D_i(t) + \sum_{j<i}H^\text{int}_{ij} \right), 
+
+where :math:`H^D_i` is the driving Hamiltonian for atom :math:`i` and
+:math:`H^\text{int}_{ij}` is the interaction Hamiltonian between atoms :math:`i`
+and :math:`j`. Note that, if multiple basis are addressed, there will be a 
+corresponding driving Hamiltonian for each transition.
+
+
+Driving Hamiltonian
+*********************
+
+The driving Hamiltonian describes the coherent excitation of an individual atom
+between two energies levels, :math:`|a\rangle` and :math:`|b\rangle`, with
+Rabi frequency :math:`\Omega(t)`, detuning :math:`\delta(t)` and phase :math:`\phi(t)`.
```

Comment:
I would put in the body text that b is the state of higher energy, not only in the warning below.
Also, a figure of the energy levels annotated with Ω, δ etc would make everything super clear.

```suggestion
The driving Hamiltonian describes the coherent excitation of an individual atom
between two energies levels, :math:`|a\rangle` and :math:`|b\rangle`, :math:`E_b > E_a`, with
Rabi frequency :math:`\Omega(t)`, detuning :math:`\delta(t)` and phase :math:`\phi(t)`.
```

Reply:
If I add the picture, do you think it's worth it to repeat in the text that the energy of b is higher than a? I feel like the picture and the warning should be enough

Reply:
I'd rather yes, I add to re-read three time to make sure about the convention between an and b. The warning block had the unintended effect that, being a block of its own, I expected to find only _additional_ information, not the required ones to understand the body text 😅
In fact the warning is really here to insist on the ordering despite the vector space representation, so the energy ordering should be stated in the body text I think.

Reply:
(Long comment for a nit, just merge <3 )